diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
-```
+curl --request GEt http://www.example.com/donuts```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function() {
+$.ajax({
+  url: 'http://www.example.com/donuts',
+  type: 'GET'
+})
+.done(logResponseBody)
```

Comment:
i know you're not using a linter here but, a reminder that semicolons are important.